converters/fromZigbee.js

```diff
@@ -3818,6 +3818,74 @@ const converters = {
             }
         },
     },
+    danfoss_thermostat_att_report: {
+      cluster: 'hvacThermostat',
+      type: ['attributeReport', 'readResponse'],
+      convert: (model, msg, publish, options, meta) => {
+          const result = {};
+          if (typeof msg.data['localTemp'] == 'number') {
+              result[postfixWithEndpointName('local_temperature', msg, model)] = precisionRound(msg.data['localTemp'], 2) / 100;
+          }
+          if (typeof msg.data['occupiedHeatingSetpoint'] == 'number') {
+              let ohs = precisionRound(msg.data['occupiedHeatingSetpoint'], 2) / 100;
+              // Stelpro will return -325.65 when set to off
```

Comment:
Seems to be copied from a different device, so maybe update the comment... if it's exactly the same aside from a few values you can also do this... which is cleaner.

https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/converters/fromZigbee.js#L3892
It uses the thermostat_att_report convertor and then adds deviating properties below and fixes up some existing ones that may not be 100% how you expect. It saves on maintenance and code duplication.

Reply:
I'll review the setup on the weekend and verify it works, then update the PR